bindings/go/context_params.go

```diff
@@ -18,15 +18,15 @@ import "C"
 // semantics UCP library may avoid allocation of expensive resources associated with
 // send/receive queues.
 type UcpParams struct {
-	params C.ucp_params_t
+	contextParams C.ucp_params_t
```

Comment:
why renamed?

Reply:
to be clear from ucpParams structure. Like not have `params.params`